Governance/retire/retire-replica.md

```diff
@@ -19,24 +19,24 @@ Some methods are machine-driven, where you use any available statistics automati
 
 ### Community feedback
-* E-mail communication: this is possible where your community is well defined, so if users need to sign up to a group/project or if there are well defined community of potential users. For example, if the data is on NCI, you can use my.nci.org.au to contact projects' users. This might be sufficient when you have already identified a likely candidate for removal.
-* Survey: this can be useful when you need to evaluate several datasets at one time or work out which subset of a big dataset could be retired.
-* Project dependence: if a dataset was replicated as part of a particular research project, then it can be considered for removal on completion of that project unless input data is required for reproducibility for some specified period.
-* ...??? 
+* **E-mail communication:** this is possible where your community is well defined, so if users need to sign up to a group/project or if there are well defined community of potential users. For example, if the data is on NCI, you can use my.nci.org.au to contact projects' users. This might be sufficient when you have already identified a likely candidate for removal.
+* **Survey:** this can be useful when you need to evaluate several datasets at one time or work out which subset of a big dataset could be retired.
+* **Project dependence:** if a dataset was replicated as part of a particular research project, then it can be considered for removal on completion of that project unless input data is required for reproducibility for some specified period.
 
 ### Suggested procedure
 
-1. Using one or more automated methods to collect statistics on actual data usage.
+1. Use one or more automated method(s) to collect statistics on actual data usage.
 2. Announce to users with data access the anticipated retirement of the dataset. If the data appears to be unused, move to a "quarantine" area nearby where users may still find the data and notify the data managers that it is still required.
-3. If no objection is heard, the data can be staged for deletion. On the other hand, if objection is heard, then a dialogue must be started with the affected users to identify an alternative support mechanism for the dataset.
-    a. An alternative mechanism to save disk space while data is still required for infrequent access is to maintain the dataset on tape storage
+3. If no objection is heard, the data can be staged for deletion (moved to a "for deletion" location and read access to users removed). On the other hand, if objection is heard, then a dialogue must be started with the affected users to identify an alternative support mechanism for the dataset.</br>
+``` {retire}
+NOTE An alternative mechanism to save disk space while data is still required for infrequent access is to maintain the dataset on tape storage
+```
```

Comment:
@paolap this worked in preview but it looks like the "note" is not rendering in the final book?

Reply:
The note is not rendered and the numbering following is broken. Please feel free to maybe 'edit in production' if you can to fix it? maybe using a > blockquote instead?